deploy/config/local/server.conf

```diff
@@ -15,7 +15,6 @@ upstream php {
 
 server {
     listen  8080 default_server; # For default requests.
-    listen  80;                  # For localhost loopback only.
```

Comment:
Just to note, this was added for: 
https://github.com/ministryofjustice/justice-gov-uk/blob/main/docker-compose.yml#L21C9-L22C35